solr/core/src/java/org/apache/solr/servlet/CoreContainerProvider.java

```diff
@@ -226,6 +229,21 @@ public void init(ServletContext servletContext) {
         StartupLoggingUtils.changeLogLevel(logLevel);
       }
 
+      // Do initial logs for experimental Lucene classes.
+      // TODO: Use "MethodHandles.lookup().ensureClassInitialized()" instead of "Class.forName()"
+      //   once JDK 15+ is mandatory
+      Stream.of(MMapDirectory.class, VectorUtil.class)
+          .forEach(
+              cls -> {
+                try {
+                  Class.forName(cls.getName());
+                } catch (ReflectiveOperationException re) {
+                  throw new SolrException(
+                      ErrorCode.SERVER_ERROR, "Could not load Lucene class: " + cls.getName());
+                }
+              });
+      VectorUtil.dotProduct(new byte[0], new byte[0]);
```

Comment:
If the class is initialized (what the previous code does), this call is obsolete.

Reply:
My bad, I thought I had deleted that. Will remove.